OutOfSchool/OutOfSchool.BusinessLogic/Services/ProviderServices/ProviderService.cs

```diff
@@ -300,11 +300,11 @@ public async Task<ProviderDto> GetByUserId(string id, bool isDeputyOrAdmin = fal
 
         if (isDeputyOrAdmin)
         {
-            var providerAdmins = await providerAdminRepository.GetByFilter(p => p.UserId == id).ConfigureAwait(false);
-            var providerAdmin = providerAdmins.FirstOrDefault();
-            if (providerAdmin != null)
+            var employees = await employeeRepository.GetByFilter(p => p.UserId == id).ConfigureAwait(false);
```

Comment:
Need to rename `isDeputyOrAdmin` argument so it reflects new role :)

Reply:
Renamed to `isEmployee`